recipes/searchgui/build.sh

```diff
@@ -22,3 +22,6 @@ ln -sf "$(which metamorpheus)" "$outdir"'/resources/MetaMorpheus/metamorpheus'
 # removing makeblast prebuilt binary and replacing it by blast package ones
 rm -f "$outdir"'/resources/makeblastdb/linux/linux_64bit/makeblastdb'
 ln -sf "$(which makeblastdb)" "$outdir"'/resources/makeblastdb/linux/linux_64bit/makeblastdb'
+
+# removing MsAmanda prebuilt binary for macosx until .NET Core 6 is released
+rm -f "$outdir"'/resources/MS Amanda/osx/*'
```

Comment:
This didn't work because the `*` glob was in the quoted part
```suggestion
rm -f "$outdir"'/resources/MS Amanda/osx/'*
```
or just explicitly
```suggestion
rm -f "$outdir"'/resources/MS Amanda/osx/MSAmanda'
```
should work.
(As for the `install_name_tool` issue, I have no idea though.)

Reply:
Thank you very much, I will try it...